### ISSUE TITLE
The Lab landing page shows white text on a white background

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#6374]
## Expected behavior
The lab landing page should show white text against a dark background. If the background image errors or fails to load, the page background be a dark colour as a fallback.

## Current behavior
<img width="1248" alt="The lab landing page with a white background on white text." src="https://user-images.githubusercontent.com/59547/214061219-134dae42-ecd8-4270-8b82-4e1beb32d5d6.png">

The background image isn't loading and the background colour is set to the same colour as the page text.

## Steps to replicate
https://www.zooniverse.org/lab while not logged in.
